sprite.go

```diff
@@ -731,6 +738,9 @@ func (p *Sprite) goAnimate(name string, ani *aniConfig) {
 		}
 	})
 	waitToDo(animwg.Wait)
+	if isNeedPlayDefault {
+		p.playDefaultAnim()
```

Comment:
If there is no default animation (there is default costume). We should use the default costume?